ibc_public/utils_data.py

```diff
@@ -41,17 +41,18 @@
                                      CONTRASTS['right label'][i]]
 
 
-def get_subject_session(protocol):
+def get_subject_session(*protocols):
```

Comment:
The argument should be `protocols` not `*protocols`

Reply:
Done. Is there any problem with the *args syntax? I find it more flexible since that way you are not forced to pass a list of one element when you want to pass only one argument